### PR TITLE
Change benchmarks --> examples in code

### PR DESCRIPTION
**Pull request by nik-mosaic**
Code (& YAMLs) should reflect repository name change.

### CODE REVIEW

**Review by vchiley — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# benchmarks
+# examples
```
Comment (dakinggg) [inline README.md:1]: I know it wasn't before, but this should probably be capitalized?

**Review by dakinggg — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (nik-mosaic) [reply README.md]: Sure.

**Review by dblalock — APPROVED** [review]

**Pull request merged**